homeassistant/components/openhome/media_player.py

```diff
@@ -123,6 +126,21 @@ def turn_off(self):
         """Put device in standby."""
         self._device.SetStandby(True)
 
+    def play_media(self, media_type, media_id, **kwargs):
+        """Send the play_media command to the media player."""
+        if not media_type == MEDIA_TYPE_MUSIC:
+            _LOGGER.error(
+                "Invalid media type %s. Only %s is supported",
+                media_type,
+                MEDIA_TYPE_MUSIC,
+            )
+            return
+        track_details = {
+            "title": "Home Assistant"
```

Comment:
```suggestion
            "title": "Home Assistant",
```

Reply:
Oops - that's what I get for copy and paste without thought...

Reply:
No, it is my fault 😁 forgetting something like that so easily.